main.py

```diff
@@ -307,6 +319,13 @@ def hotkey(self, event):
         elif event.char == "z":
             print("Deleting last box...");
             self.frame_canvas.clear_last_label()
+        elif event.char == "r":
+            self.frame_canvas.rollover_label = True
+            print(f"r pressed rollover is now {self.frame_canvas.rollover_label}")
+        elif event.char == "t":
+            self.frame_canvas.rollover_label = False
```

Comment:
should we use just 1 hotkey for toggling? self.frame_canvas.rollover_label = not self.frame_canvas.rollover_label

Reply:
I second this, I think a one key toggle would be good